src/base/tokenizer/tokenizer.ts

```diff
@@ -36,7 +36,7 @@ export class AnyTokenizer implements ITokenizer {
 		const vscode = await import('vscode');
 		return this.countTokens({
 			role: this.toChatRole(message.role),
-			content: [new vscode.LanguageModelTextPart(message.content)],
+			content: [new vscode.LanguageModelTextPart(message.content as string)],
```

Comment:
How are images handled here? This could throw some errors if we just assume it's a string and give it to VS Code API